arches/install/arches_admin.py

```diff
@@ -135,6 +135,7 @@ def handle(self, options):
             ".coveragerc",
             "pyproject.toml",
             ".pre-commit-config.yaml",
+            ".github/workflows/main.yml",
```

Comment:
I think this will create mixed slashes on Windows, but I see we have this problem in several places already, so I'll open a new issue for it.